README.md

```diff
@@ -9,6 +9,25 @@ This repository is becoming a backend for what will eventually become presentati
 
 Thanks for your patience!
 
+### Submission Process
+This is inprogress and subject to change, please feel free to PR ideas and changes you'd like to see:
+
+> What kind of talks are accepted v. Won’t be accepted?
+
+For now we are concentrating on talks given at CNCF Events since those have already been accepted by the community. We don't want to do editorializing in this repo, if it's already been given at a CNCF event submit it. 
```

Comment:
```suggestion
We are currently focusing on talks presented at CNCF Events, as these have already been vetted and accepted by the community. We aim to avoid editorializing in this repository. If your talk has been presented at a CNCF event, feel free to submit it.
```